cmd/nodeadm/debug/debug.go

```diff
@@ -92,6 +93,8 @@ func (c *debug) Run(log *zap.Logger, opts *cli.GlobalOptions) error {
 	runner := validation.NewRunner[*api.NodeConfig](printer)
 	apiServerValidator := node.NewAPIServerValidator(kubelet.New())
 	clusterProvider := kubernetes.NewClusterProvider(awsConfig)
+
+	runner.Register(hybrid.NewHybridNodeConfigValidator(nodeConfig, log))
```

Comment:
lets just remove this from debug for now, per previous discussions with @g-gaston. debug is not meant to be run until init has succeeded.  If a user does run debug before getting a successful init, most of the checks are going to return back errors because its expecting the node to be setup and working.  Validating the config adds little value in those cases.  

Its a minor change either way and if we see customer usage of debug/init not matching our expectations and it makes sense to add this back, we always can later.